packages/SA/src/outputTools.ts

```diff
@@ -0,0 +1,99 @@
+// outputTools.ts
+
+import * as grok from 'datagrok-api/grok';
+import * as ui from 'datagrok-api/ui';
+import * as DG from 'datagrok-api/dg';
+
+const UNSUPPORTED_TYPE_ERROR_MSG = 'Unsupported output type.';
+
+type OutputInfo = {
+  name: string,  
+  type: DG.TYPE
+};
+
+function getOutputsInfo(func: DG.Func): OutputInfo[] {
+  const ouputInfo = Array<OutputInfo>(0);
+
+  for (const item of func.outputs)
```

Comment:
Could be replaced in one-line by `map` function

Reply:
Done

Reply:
GREAT THANKS FOR CODE REVIEW!